### PR TITLE
Swap To Postgresql

### LINKED ISSUE #353: Swap to Postgesql

### PR DESCRIPTION
**Pull request by Pdzly**
Closes #353 

Docker changes:
https://github.com/sublinks/sublinks-docker/tree/feature/api-353-swap-to-postgres

### CODE REVIEW

**Review by jgrim — DISMISSED** [review]

**Comment by jgrim** [comment]
We can merge this in once the docker configuration supports it.

**Comment by DarkArc** [comment]
There seem to be a fair number of odd whitespace changes and some imports that aren't relevant.

It's not a huge deal, but it would be nice to clean those up.

**Comment by Pdzly** [comment]
I will wait and see if federation swaps over to postgresql too, just for organisational side.

**Review by jgrim — DISMISSED** [review]

**Review by lazyguru — CHANGES_REQUESTED** [review]

**Review by jgrim — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -11,30 +11,28 @@ updates:
     schedule:
       interval: "weekly"
   # Maintain dependencies for Gradle
-  - package-ecosystem: "gradle"
```
[the remark this answers is not in the document]
Comment (Pdzly) [reply .github/dependabot.yml:14]: This f*cked up the automatic checks. As this is in terms, illegal.

[the remark this answers is not in the document]
Comment (lazyguru) [reply .github/dependabot.yml:14]: I don't understand what you mean here? By removing this, you remove automatic dependency management of Java packages by Dependabot. Please add it back

**Pull request merged**